ext/sysvsem/sysvsem.stub.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/** @return resource|false */
+function sem_get(int $key, $max_acquire = 1, $perm = 0666, $auto_release = 1) {}
```

Comment:
```suggestion
function sem_get(int $key, int $max_acquire = 1, int $perm = 0666, int $auto_release = 1) {}
```

Maybe add a TODO to make `$auto_release` a bool argument, that would make more sense.